spec/core_functions/color/change_color/error/bounds.hrx

```diff
@@ -206,6 +308,39 @@ Error: argument `$lightness` of `change-color($color, $red: false, $green: false
 
    ------^
 
+<===>
+================================================================================
+<===> lightness/unit/input.scss
+// This test covers sass/dart-sass#1745, but should be removed once units are
+// fully forbidden (sass/sass#3374).
+a {b: change-color(red, $lightness: 200px)}
+
+<===> lightness/unit/error
+DEPRECATION WARNING: $lightness: Passing a number without unit % (200px) is deprecated.
```

Comment:
I feel like "a number without unit %" may sound strange to users who may not realize % is considered a unit in sass

Edit: I guess that's out of scope for this change

Reply:
Yeah, I think this is something we just have to expect users to learn at some point.